README.md

```diff
@@ -95,6 +95,8 @@ txs = [
 appsSdk.sendTransactions(txs);
 ```
 
+> Note: `value` accepts a number or a string as a decimal number (hexadecimal is not supported).  
```

Comment:
where is this coming from 🤔 I though we just sync a revert (which should not trigger any changes only update the commit history)

Reply:
oops